packages/core/src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Tab, withStyles, Theme } from '@material-ui/core';
+import { BackstageTheme } from '@backstage/theme';
+
+const withStylesProps = (styles: any) => (Component: any) => (props: any) => {
```

Comment:
This is already built into MUI, see https://material-ui.com/styles/api/#makestyles-styles-options-hook for how to access props inside makeStyles